setup.py

```diff
@@ -1,7 +1,4 @@
 #!/usr/bin/env python
 from setuptools import setup
 
-
-setup(
-
-)
+setup()
```

Comment:
@CyclingNinja This change should also be fixed in the template. rendering with a blank line like that is gross.

Reply:
Yeah agreed, I'm wondering how it even happened